### PR TITLE
fix: add homepage loader

### PR DESCRIPTION
**Pull request by DmitriyGo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
       href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="src/styles/preloadedStyles.css">
```
Comment (DmitriyGo) [inline index.html:14]: idk how to do it with styled-components
createGlobalStyles works only after the scripts are loaded, but before that the white background is displayed

Comment (yarre-uk) [reply index.html:14]: damn man, idk either

File: src/shared/components/Loaders/FullPageLoader/FullPageLoader.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { PageLoaderWrapper } from './FullPageLoaderStyles';
+import CircleLoader from '../CircleLoader/CircleLoader';
+
+const FullPageLoader = ({}) => {
```
Comment (yarre-uk) [inline src/shared/components/Loaders/FullPageLoader/FullPageLoader.tsx:4]: remove useless parentheses

**Comment by turchin-aa** [comment]
That's nice you created a few loaders by yourself but I don't see any reasons for it. You could just find any svg loader on the Internet and copy it to our project.

**Review by turchin-aa — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (turchin-aa) [reply index.html:14]: That's ok.

**Comment by yarre-uk** [comment]
He just copied them from internet either

**Review by yarre-uk — APPROVED** [review]

**Comment by turchin-aa** [comment]
> He just copied them from internet either

Ok man.

**Pull request merged**